docs/source/concepts/ledger-model/ledger-structure.rst

```diff
@@ -153,7 +151,7 @@ In practice, multiple IOU contracts can exist between the same `obligor` and
 in this section, each contract only appears once, allowing us to drop the notion 
 of identifiers for simplicity reasons.
 
-A **transaction** is a list of actions. Thus, the consequences of
+A **transaction** is a non-empty list of actions. Thus, the consequences of
```

Comment:
i don’t think we need this restriction in the ledger model, I’d omit it.

Reply:
I think it should be either here, or a bit later:
"a proper subtransaction: a **non-empty** transaction obtained by removing at least one action, or"
the original definition maks an empty transaction a proper subtransaction, which is not I guess.
What do you think?

Reply:
What’s the issue with making an empty transaction a proper subtransaction?

Reply:
All this terminology is borrowed from mathematics, where `sub-X` and `proper sub-X` have a fairly consistent meaning. And there the empty transaction would be considered a proper subtransaction of any non-empty transaction.

Reply:
Then maybe the image is wrong?
"the image below shows all its proper subtransactions"
But the empty transaction is not on the image.
How about we add that too? An empty yellow box?

Reply:
Yeah, I'd be fine with relabelling the image to the "image below shows all its proper non-empty subtransactions".